parsons/utilities/dbt/dbt.py

```diff
@@ -0,0 +1,126 @@
+"""Core methods for running dbt commands."""
+
+import logging
+import pathlib
+from typing import List, Optional, Type, Union
+
+from dbt.cli.main import dbtRunner, dbtRunnerResult
+
+from parsons.utilities.dbt.logging import dbtLogger
+from parsons.utilities.dbt.models import Manifest
+
+logger = logging.getLogger(__name__)
+
+
+class dbtRunnerParsons:
+    def __init__(
+        self,
+        commands: Union[str, List[str]],
+        dbt_project_directory: pathlib.Path,
+    ) -> None:
+        """Initialize dbtRunner with commands and a working directory.
+
+        `Args:`
+            commands: Union[str, List[str]]
+                A single dbt command string or a list of dbt command
+                strings.
+                e.g. ["seed", "build -s models/staging", "test"]
+            dbt_project_directory: pathlib.Path
+                The path to find the dbt project, as a working
+                directory for dbt commands to run
+        """
+        if isinstance(commands, str):
+            commands = [commands]
+        self.commands = commands
+        self.dbt_project_directory = dbt_project_directory
+
+    def run(self) -> list[Manifest]:
+        """Executes dbt commands one by one, returns all results."""
+        results = []
+
+        for command in self.commands:
+            result = self.execute_dbt_command(command)
+            results.append(result)
+
+        return results
+
+    def execute_dbt_command(self, command: str) -> Manifest:
+        """Runs dbt command and logs results after process is completed.
+
+        If raise_error is set, this method will raise an error if the dbt
```

Comment:
Where is raise_error set? Where is it referenced? Is this an underlying dbt thing or is it part of this method/class?